examples/kyb-app/src/common/utils/is-cuid.ts

```diff
@@ -0,0 +1,7 @@
+export const isCuid = (str: unknown): str is string => {
+  if (typeof str !== 'string') return false;
+
+  const CUIDRegExp = /^[a-z0-9]{7,}$/;
```

Comment:
Note CUIDs have a very specific max length. Usually it's recommended to use the `cuid`'s package `isCuid` util for this. Still unsure if we need to know about the id on the frontend.